File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio10.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio10 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira a altura da parede: ");
        float altura = entrada.nextFloat();
        System.out.println("Insira a largura da parede: ");
        float largura = entrada.nextFloat();

        float area = altura * largura;
        System.out.println("Área a ser pintada: " +area);
        float tinta = area / 2;
        System.out.println("Latas de tinta: " +tinta);
    }
}

File: guanabara-c/lista01/exercicio10.c
#include <stdio.h>

int main(){
    float altura, largura, area, tinta;

    printf("Insira a altura da parede: ");
    scanf("%f", &altura);
    printf("Insira a largura da parede");
    scanf("%f", &largura);

    area = altura * largura;
    tinta = area / 2;
    printf("Área a ser pintada: %.2f m².\nLitros de tinta necessários: %.2f litros", area, tinta);

    return 1;
}
